service_mesh/v2x/ossm-custom-resources.adoc

```diff
@@ -27,10 +27,21 @@ include::modules/ossm-cr-pilot.adoc[leveloffset=+2]
 
 include::modules/ossm-configuring-kiali.adoc[leveloffset=+1]
 
-include::modules/ossm-configuring-jaeger.adoc[leveloffset=+1]
 
-include::modules/ossm-jaeger-config-elasticsearch.adoc[leveloffset=+2]
+== Configuring Distributed Tracing
```

Comment:
I'm not sure that configuring distributed tracing describes what this is about. Is this use case about configuring an external Jaeger instance? If so, and I have an external Jaeger instance how do I search for that? I guess I'm a little confused because if you're using the default Jaeger, there isn't much configuration to do. For the external Jaeger scenario, there is more configuration but I don't see any of those keywords. Connecting to an existing Jaeger instance? Configuring distributed tracing outside of OSSM? 

If they want a default Jaeger config, they have it already from the installation docs. I think this heading should more specifically speak to what steps you need to take if you want to set up an external instance. 

I also think this might not be best in the custom resources assembly. If it's stepwise configuration information it might be better in the configuration assembly.

Reply:
Now I'm confused, we no longer have a configuration assembly in OSSM, we moved that information into the Custom Resource assembly.  

Yes the overall use case  for this PR is how to point to an external Jaeger instance, which includes updating how to configure Jaeger in the new SMCP and pointing out that you MUST point to an external Jaeger instance for a streaming deployment (because you cannot configure streaming in the new SMCP).  The file that actually supports the "external jaeger" use case is ossm-specifying-jaeger-configuration.  Everything else in this PR is updating the Jaeger configuration docs for the SMCP v2.0.

Reply:
Maybe instead of "external jaeger," we should use, "user managed jaeger."

Reply:
I think that configuration documentation should go in the configuration assembly. I moved this content to the custom resources assembly because it seemed more like reference documentation than configuration documentation.

I understand this is updating something that already exists. I'm not sure the content about how to enable the default Jaeger needs more explanation here. I think it would be stronger if it focused on the user managed jaeger content.

Reply:
Just like the rest of the product, the default Jaeger configuration is NOT suitable for production and must be configured before you deploy to a production environment.  Which is why I didn't understand why you considered the configuration information "reference" material.  But at the time, pulling that information out of the Customizing the Installation assembly wasn't something I was completely opposed to, as long as the reference section came after the Customization assembly (which it doesn't).  If we're going to revist how we present the configuration and deployment information, I'd really like to do that in another PR, because the Jaeger configuration information in the docs has been incorrect ever since the v2 release on Nov 4.  I really just want to get this information published.